### DISCUSSION TITLE
Detalle con requerimiento Mapa de heladeras en parte 2

### DISCUSSION THREAD

**Discussion by D-M-Gonzalez** [dds-jv/foro#432] [TPA - 2024] [answered]
Buenas! es un detalle nada mas, que me parece se paso por alto:

El mapa de heladeras deberá ser capaz de mostrar, junto a cada heladera su nivel de llenado así como si necesita atención. Una heladera necesita atención cuando las últimas 3 lecturas de temperatura dieron por **debajo del mínimo configurado**. 

En la mayoría de los casos ( fallas del compresor, pinchaduras, perdida de gas, falla eléctrica, sobrecarga de alimentos, etc. ), cuando una heladera presenta fallas la temperatura sube, y que suba es peligroso porque los alimentos pueden perder la cadena de frio. Que la medición de más frío que el mínimo suele ser porque alguien le dio demás al termostato o este se quedo pegado.

Pero eso, me pareció raro que el requerimiento hable de falla si la temperatura es menor al mínimo, que como mucho te congela los alimentos pero rara vez los hecha a perder, y no del máximo que es lo realmente peligroso y en unos minutos te puede hechar a perder todo.

**Comment by flbulgarelli** [comment, marked as answer]
Es correcto como lo interpretaste.  ¡Saludos!
